omnibus/config/software/libxcrypt.rb

```diff
@@ -0,0 +1,40 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Is there a reason why this shouldn't live in https://github.com/DataDog/omnibus-software?

Reply:
I've thought about this earlier, and the reason I've put this here is that this is a dependency of the `python2` and `python3` software definitions that live in this repository, not of the `python2` and `python3` software definitions that live in omnibus-software (the `datadog-agent` definitions hide the `omnibus-software` ones).

I'm not confident with adding `libxcrypt` as a dependency of the `omnibus-software` definitions because we're not testing whether this `libxcrpyt` build works with them, since we don't use them (they use older [Python 2](https://github.com/DataDog/omnibus-software/blob/master/config/software/python2.rb) and [Python 3](https://github.com/DataDog/omnibus-software/blob/master/config/software/python3.rb) versions). Therefore, I don't think the changes belong in the shared omnibus-software catalog as long as the Python 2 / Python 3 definitions are not also updated there.

Reply:
Ok, I don't have a strong preference about where this should go, so I think that this is a good reason to keep it here. Thanks!